contact_map/contact_map.py

```diff
@@ -7,6 +7,9 @@
 import itertools
 import pickle
 import json
+
+import warnings
+
```

Comment:
Any reason for the excess newlines around this import?

Reply:
reminder that this is temporary and should be removed